# Translation of Odoo Server.
# This file contains the translation of the following modules:
#	* auth_signup_verify_email
#
msgid ""
msgstr ""
"Project-Id-Version: Odoo Server 12.0\n"
"Report-Msgid-Bugs-To: \n"
"PO-Revision-Date: 2021-08-15 16:49+0000\n"
"Last-Translator: JonathanStein <jstein@image.dk>\n"
"Language-Team: none\n"
"Language: da\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: \n"
"Plural-Forms: nplurals=2; plural=n != 1;\n"
"X-Generator: Weblate 4.3.2\n"

#. module: auth_signup_verify_email
#: code:addons/auth_signup_verify_email/controllers/main.py:69
#, python-format
msgid "Another user is already registered using this email address."
msgstr "En anden bruger er allerede registreret med denne mailadresse."

#. module: auth_signup_verify_email
#: code:addons/auth_signup_verify_email/controllers/main.py:79
#, python-format
msgid "Check your email to activate your account!"
msgstr "Kig i din mail for at aktivere din konto!"

#. module: auth_signup_verify_email
#: code:addons/auth_signup_verify_email/controllers/main.py:74
#, python-format
msgid "Something went wrong, please try again later or contact us."
msgstr "Noget gik galt. Forsøg venligst igen senere eller kontakt os."

#. module: auth_signup_verify_email
#: code:addons/auth_signup_verify_email/controllers/main.py:40
#, python-format
msgid "That does not seem to be an email address."
msgstr "Dette ser ikke ud til at være en gyldig mailadresse."
